File: Survey Project/Survey Project/New Question Screen/NewQuestion.swift
//
//  NewQuestion.swift
//  Survey Project
//
//  Created by Daniel Butrashvili on 22/12/2018.
//  Copyright © 2018 Daniel Butrashvili. All rights reserved.
//

import Foundation

class NewQuestion : Encodable{
    
    var question : String
    var categoryName : String
    var answers : [String]
    var askerToken : String
    
    init(){
        question = ""
        answers = []
        categoryName = ""
        askerToken = Auth.userToken
    }
    
    
    
}

File: Survey Server/src/Entities/NewQuestion.java
package Entities;

import java.util.List;

public class NewQuestion {
    String question;
    String categoryName;
    String askerToken;
    List<String> answers;

    public NewQuestion(String question, String categoryName, List<String> answers,String askerToken) {
        this.question = question;
        this.categoryName = categoryName;
        this.answers = answers;
        this.askerToken = askerToken;
    }

    public String getQuestion() {
        return question;
    }

    public String getCategoryName() {
        return categoryName;
    }

    public List<String> getAnswers() {
        return answers;
    }

    public String getAskerToken() {
        return askerToken;
    }
}
